dask_sql/utils.py

```diff
@@ -62,12 +62,12 @@ class Pluggable:
     __plugins = defaultdict(dict)
 
     @classmethod
-    def add_plugin(cls, name, plugin, replace=True):
+    def add_plugin(cls, names: List[str], plugin, replace=True):
```

Comment:
Wondering if we want to allow both `str` and `List[str]` and just have handling for the string case like

```python
if isinstance(names, str):
    names = [names]
```

That would greatly simplify the diff here

Reply:
Yes, that is a much better approach. Will make the change.